src/connection.js

```diff
@@ -392,7 +393,23 @@ export default class Connection extends EventEmitter {
     if (this.config.options.localAddress) {
       connectOpts.localAddress = this.config.options.localAddress;
     }
-    this.socket.connect(connectOpts);
+    
+    if (!multiSubnetFailover) {
+        this.socket.connect(connectOpts);
+    }
+    else {
```

Comment:
Can you put the closing `}` and the `else {` on the same line, as in other areas of the existing code base?